mmf/configs/models/unit/defaults.yaml

```diff
@@ -0,0 +1,104 @@
+model_config:
+  unit:
+    detr_ckpt_path: ''  # the initial DETR parameters to start from (this should be a checkpoint file trained w/ DETR package)
+    detr_ckpt_load_backbone_only: false
+    detection_loss_weight: 1.
+
+    keep_only_bert_cls:
+      # whether to keep only the [CLS] token's hidden states in transformer
+      vl:
```

Comment:
`unit_ckpt_load_backbone_only` or `ckpt_load_backbone_only`.

Reply:
renamed to `base_ckpt_load_backbone_only`